script/core

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+# Helper to start Home Assistant Core inside the devcontainer
+
+# Stop on errors
+set -e
+
+if [ -z "${DEVCONTAINER}" ]; then
+  echo "This task should only run inside a devcontainer, for local install HA Core in a venv."
+  exit 1
+fi
+
+if [ ! -z "${CODESPACES}" ]; then
+  WORKSPACE="/root/workspace/frontend"
+else
+  WORKSPACE="/workspaces/frontend"
+fi
+
+if [ -z $(which hass) ]; then
+  echo "Installing Home Asstant core from dev."
+  python3 -m pip install --upgrade \
+    colorlog \
+    git+git://github.com/home-assistant/home-assistant.git@dev
+fi
+
+if [ ! -d "${WORKSPACE}/config" ]; then
+  echo "Creating default configuration."
+  mkdir -p "${WORKSPACE}/config";
+  echo "default_config:
```

Comment:
I actually meant with my comment, shouldn't we use the `ensure_config` check and then append `demo:` + `logger:` ?

Reply:
That way we can also drop the `touch` at the end of this file.

Reply:
Ah!
Did not know of that script (looks like it's broken (at least on python 3.9))